src/config/index.ts

```diff
@@ -376,11 +382,40 @@ if (acgConfig) {
         port: clowdAppConfig.kafka.brokers[0].port.toString()
     };
 
+    data.kafka.topics = {
+    // Kafka topic settings
+        advisor: {
+            topic: processTopicName(clowdAppConfig, "platform.remediation-updates.advisor"),
+            enabled: processTopicEnabled(clowdAppConfig, "platform.remediation-updates.advisor"),
+        },
+        compliance: {
+            topic: processTopicName(clowdAppConfig, "platform.remediation-updates.compliance"),
+            enabled: processTopicEnabled(clowdAppConfig, "platform.remediation-updates.compliance"),
+        },
+        inventory: {
+            topic: processTopicName(clowdAppConfig, "platform.inventory.events"),
+            enabled: processTopicEnabled(clowdAppConfig, "platform.inventory.events"),
+        },
+        patch: {
+            topic: processTopicName(clowdAppConfig, "platform.remediation-updates.patch"),
+            enabled: processTopicEnabled(clowdAppConfig, "platform.remediation-updates.patch"),
+        },
+        receptor: {
+            topic: processTopicName(clowdAppConfig, "platform.receptor-controller.responses"),
+            enabled: processTopicEnabled(clowdAppConfig, "platform.receptor-controller.responses"),
+        },
+        vulnerability: {
+            topic: processTopicName(clowdAppConfig, "platform.remediation-updates.vulnerability"),
+            enabled: processTopicEnabled(clowdAppConfig, "platform.remediation-updates.vulnerability"),
+        }
+    }
+
     if (_.get(clowdAppConfig, 'kafka.brokers[0].sasl', '') !== '') {
         data.kafka.sasl = {
             username: clowdAppConfig.kafka.brokers[0].sasl.username,
             password: clowdAppConfig.kafka.brokers[0].sasl.password,
             mechanism: clowdAppConfig.kafka.brokers[0].sasl.saslMechanism,
+            securityProtocol: clowdAppConfig.kafka.brokers[0].sasl.securityProtocol,
```

Comment:
@psav IIRC the securityProtocol field doesn't exist with kafkaJS, does it actually exist in the library?

Reply:
This isn't their library data struct :) This is our config struct